packages/ui/src/components/TextField/index.tsx

```diff
@@ -61,14 +69,17 @@ export const TextField = forwardRef<
               fontColor={fontColor}
             />
           ) : (
-            <StyledInput
-              {...props}
-              ref={ref as ForwardedRef<HTMLInputElement>}
-              placeholder={placeholder}
-              spellCheck={false}
-              isAdmin={isAdmin}
-              fontColor={fontColor}
-            />
+            <InputContainer>
+              <StyledInput
+                {...props}
+                ref={ref as ForwardedRef<HTMLInputElement>}
+                placeholder={placeholder}
+                spellCheck={false}
+                isAdmin={isAdmin}
+                fontColor={fontColor}
```

Comment:
```suggestion
                fontColor={fontColor}
                isRight={right ? true : false}
```

요기서 내부 스트링이 너무 길어지면 아이콘이랑 영역이 겹쳐서, padding 설정을 다르게 해주면 좋을 것 같아요!!!
더 좋은 방법이 있다면... 반영 부탁드려요🙇🏻‍♀️

<img width="343" alt="image" src="https://github.com/CEOS-Developers/CEOS-FE/assets/65931227/1a6505dd-dec6-42d4-90e8-c02c1967f8f5">